index.html

```diff
@@ -370,9 +370,15 @@ <h3>Ecosystem Overview</h3>
           <dd>
 A role an [=entity=] performs by asserting [=claims=] about one or
 more [=subjects=], creating a [=verifiable credential=] from these
-[=claims=], and transmitting the [=verifiable credential=] to a
-[=holder=]. Example issuers include corporations, non-profit organizations,
+[=claims=], and transmitting the [=verifiable credential=] to the
+[=issuee=]. Example issuers include corporations, non-profit organizations,
```

Comment:
Is there only one issuee, by definition? 

Or could the issuer simultaneously transmit the VC to multiple issuees?
```suggestion
[=claims=], and transmitting the [=verifiable credential=] to an
[=issuee=]. Example issuers include corporations, non-profit organizations,
```

Reply:
I believe that current protocols are 1 to 1, not 1 to many. So "the" is appropriate

Reply:
I believe that all current protocols are 1 to 1 and not 1 to many

Reply:
> I believe that all current protocols are 1 to 1 and not 1 to many

But is that a necessary restriction, to be enshrined in this standard? I think one-to-many is, and should be preserved as, a possibility.